libraries/rtos/rtx/TARGET_CORTEX_M/RTX_Conf_CM.c

```diff
@@ -317,10 +355,16 @@ void os_idle_demon (void) {
  *      RTX Errors
  *---------------------------------------------------------------------------*/
 extern void mbed_die(void);
+extern osThreadId svcThreadGetId (void);
+
+#include "stdio.h"
 
 void os_error (uint32_t err_code) {
     /* This function is called when a runtime error is detected. Parameter     */
-    /* 'err_code' holds the runtime error code (defined in RTX_Conf.h).      */
+    /* 'err_code' holds the runtime error code (defined in RTX_Config.h).      */
+    osThreadId err_task = svcThreadGetId();
+    printf("CMSIS RTX err_code: 0x%lu \r\n", err_code);
+    printf("TASK_ID = 0x%X \r\n", (int)err_task);
     mbed_die();
```

Comment:
This should be replaced with a single-line call to `error(...)` for those of us that have overridden `error(...)` with our own implementation. Example:

```
osThreadId err_task = svcThreadGetId();
error("CMSIS RTX error code: 0x%08X, task ID: 0x%08X\n", err_code, (uint32_t)err_task);
```